weave/tests/test_arrow_vectorizer.py

```diff
@@ -1008,41 +1008,42 @@ def test_arrow_dict():
     assert awl.object_type == weave.type_of(expected_output).object_type
 
 
-def test_vectorize_works_recursively_on_weavifiable_op():
-    # this op is weavifiable because it just calls add
-    @weave.op()
-    def add_one(x: int) -> int:
-        return x + 1
-
-    weave_fn = weave_internal.define_fn(
-        {"x": weave.types.Int()}, lambda x: add_one(x)
-    ).val
-    vectorized = arrow.vectorize(weave_fn)
-    expected = vectorized.to_json()
-    print("test_vectorize_works_recursively_on_weavifiable_op.expected", expected)
-    assert expected == {
-        "nodeType": "output",
-        "type": {
-            "type": "ArrowWeaveList",
-            "_base_type": {"type": "list"},
-            "objectType": "int",
-        },
-        "fromOp": {
-            "name": "ArrowWeaveListNumber-add",
-            "inputs": {
-                "self": {
-                    "nodeType": "var",
-                    "type": {
-                        "type": "ArrowWeaveList",
-                        "_base_type": {"type": "list"},
-                        "objectType": "int",
-                    },
-                    "varName": "x",
-                },
-                "other": {"nodeType": "const", "type": "int", "val": 1},
-            },
-        },
-    }
+# TODO: Temporarily commenting out this test to prevent Python unit tests from timing out and canceling
```

Comment:
nit, isn't there a `@test.skip` or something to make this a little cleaner?

Reply:
done